internal/cmd/fluxtest-harness-influxdb/test.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/cmd/flux/cmd"
+	"github.com/influxdata/flux/csv"
+	"github.com/influxdata/flux/execute/table"
+	"github.com/influxdata/flux/parser"
+	fluxClient "github.com/influxdata/influxdb/flux/client"
+	"github.com/influxdata/influxdb/tests"
+	"github.com/spf13/cobra"
+)
+
+type testExecutor struct {
+	ctx         context.Context
+	s           tests.Server
+	writeOptAST *ast.File
+	readOptAST  *ast.File
+	i           int
+}
+
+func NewTestExecutor(ctx context.Context) (cmd.TestExecutor, error) {
+	e := &testExecutor{ctx: ctx}
+	e.init()
+
+	config := tests.NewConfig()
+	config.HTTPD.FluxEnabled = true
+	config.HTTPD.FluxLogEnabled = true
+	config.Data.Index = "inmem"
```

Comment:
It'd be nice if we could test both `inmem` and `tsi1`... ideally the flux test harness would allow 1 test input to produce multiple output results.

For now, I think we'd need to push this server-setup logic into the `Run` method, and loop over the indices there. Test reporting will be a little weird if only 1 index fails for a test.